src/AzureExtension/Widgets/AzurePullRequestsDeveloperWidget.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using DevHomeAzureExtension.Client;
+using DevHomeAzureExtension.DataManager;
+using DevHomeAzureExtension.DataModel;
+using DevHomeAzureExtension.DeveloperId;
+using DevHomeAzureExtension.Helpers;
+using Microsoft.Windows.Widgets.Providers;
+using Newtonsoft.Json;
+
+namespace DevHomeAzureExtension.Widgets;
+
+internal sealed class AzurePullRequestsDeveloperWidget : AzurePullRequestsBaseWidget
+{
+    private readonly CacheManager _cacheManager;
+
+    public AzurePullRequestsDeveloperWidget()
+    : base()
+    {
+        SupportsCustomization = false;
```

Comment:
Shouldn't this widget support customization? At least to select which account it is tracking.

Reply:
It tracks all accounts that the developer has logged in, same as notifications. It's piggy backing on the same data notifications is already gathering, just showing it.